core/templates/dev/head/pages/about-page/about-page.module.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Module for the about page.
+ */
+
+import 'core-js/es7/reflect';
+import 'zone.js';
+
+import { Component, NgModule, StaticProvider } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { HttpClientModule } from '@angular/common/http';
+
+// This component is needed to force-bootstrap Angular at the beginning of the
+// app.
+@Component({
+  selector: 'service-bootstrap',
+  template: ''
+})
+export class ServiceBootstrapComponent {}
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    HttpClientModule
+  ],
+  declarations: [
+    ServiceBootstrapComponent
+  ],
+  entryComponents: [
+    ServiceBootstrapComponent
```

Comment:
Hm, there's no "about page component"? (I'm comparing with hero-detail.component.ts in the Angular upgrade guide.)

Reply:
This is the force bootstrap component and is just for the migration stage. There is no about-page.component in Angular, the About page component is still in AJS. The hero-detail.component is in Angular and hence was registered in the module.